wiki/Tournaments/GTS/KGTS_2019/en.md

```diff
@@ -0,0 +1,284 @@
+---
+tags:
+- KGTS 2019
+- KGTS2019
+---
+
+# Katacheh Global Taiko Showdown 2019
+
+![KGTS 2019 Logo](img/logo.jpg)
```

Comment:
logo missing a name? or maybe dont include the names there since they're listed below anyway. idk

Reply:
Logo works on my end. So we can resolve this (i think).

Reply:
what do u mean by this :thinking: I'm talking about the "hosted by" part on the right, it is missing Yoshii

Reply:
logo is missing yoshii cuz he wasn't host back when graphics were made

Reply:
i guess not a big deal if tournament already finished and this was used elsewhere